src/main/telemetry/crsf.c

```diff
@@ -380,7 +380,12 @@ void handleCrsfTelemetry(timeUs_t currentTimeUs)
     // in between the RX frames.
     crsfRxSendTelemetryData();
 
-    // Actual telemetry data only needs to be sent at a low frequency, ie 10Hz
```

Comment:
This comment should be retained in the appropriate place.

Reply:
whoops.. didn't mean to take this out!